src/controllers/articleController.js

```diff
@@ -92,6 +96,266 @@ class ArticleController {
       throw (err);
     }
   }
+
+  /**
+   * @description like an article
+   * @param {object} req
+   * @param {object} res
+   * @return {object} return object with a liked article
+   */
+  static async likeArticle(req, res) {
+    try {
+      const liker = req.userData.id;
+      const slugId = req.params.articleSlug;
+      const likeVote = 1;
+      const dislikeVote = 0;
+
+      const findArticle = await Article.findOne({ where: { slug: slugId } });
+
+      if (!findArticle) {
+        return res.status(200).json({
```

Comment:
200 is a success code, 404 would be much suitable here

Reply:
You're Right. I missed that! I'm gonna change it.